docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md

```diff
@@ -20,6 +20,26 @@ The Preview Requirement set includes all of the features of [Requirement set 1.8
 
 The following features are in preview.
 
+### Append on send
+
+#### [Office.Body.appendOnSendAsync](/javascript/api/outlook/office.body?view=outlook-js-preview#appendonsendasync-data--options--callback-)
+
+Added a new function to the `Body` object that appends data to the end of the item body in Compose mode.
+
+**Available in**: Outlook on Windows (connected to Office 365 subscription)
```

Comment:
Should be available on OWA too. Let's confirm from Akansha from IDC

Reply:
Confirmed offline that OWA is not in preview yet. Will add when that occurs.